src/features/presence/PresenceCollaboratorsList.scss

```diff
@@ -84,45 +84,6 @@
     font-size: 15px;
 }
 
-.bdl-PresenceCollaboratorsList-item {
-    display: flex;
-    align-items: center;
-    margin-top: 10px;
-}
-
-.bdl-PresenceCollaboratorsList-item-info-container {
-    display: inline-block;
-    margin-left: 10px;
-}
-
-.bdl-PresenceCollaboratorsList-item-info-name {
-    width: 180px;
-    overflow: hidden;
-    color: $bdl-gray;
-    font-size: 13px;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-
-    a {
-        color: $bdl-gray;
-
-        &:hover {
-            text-decoration: underline;
-        }
-    }
-}
-
-.bdl-PresenceCollaboratorsList-item-info-time {
-    color: $bdl-gray-62;
-    font-size: 12px;
-}
-
-.bdl-PresenceCollaboratorsList-list {
```

Comment:
It looks like this class is still present in `PresenceCollaboratorsList`.

Reply:
Updated!